src/oom/oomd-manager.h

```diff
@@ -3,6 +3,7 @@
 
 #include "conf-parser.h"
 #include "oomd-util.h"
+#include "sd-bus.h"
 #include "sd-event.h"
```

Comment:
actually, we generally include `sd-*.h` typically before the rest of our own headers, as kind of a test they actually work with as little else pulled in correctly. i.e. please move these two lines up